tests/validate_yaml.py

```diff
@@ -194,16 +198,148 @@ def dumper(o_filename, merged_data):
     print(f"Dumped merged data to {o_filename}")
 
 
-def help():
-    print("Usage: python validate_yaml.py -d <directory> -o <output_file>")
-    print("Options:")
-    print("-d, --dir       Directory to validate yaml files (default: config)")
-    print("-o, --output    Output file to dump merged yaml file without anchors")
-    print("-h, --help      Show this help message and exit")
-    sys.exit(1)
+def validate_rules(node, rules):
+    """
+    Validate rules for a given node
+    """
+    import kernelci.api.helper
```

Comment:
https://peps.python.org/pep-0008/
```
Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.”
```

Reply:
It is intentional, avoiding the cost of loading a heavy dependency that is used in just one rare code path.
Because kernelci.api.helper have a lot of dependencies, and this particular function of forecasting for now not planned to be used in CI, it is more manual tool to "forecast" builds. 
If i put in top, it will require to rework current workflow and include kernelci-core as dependency (and slow down whole process, making it also more fragile).